### PR TITLE
Migrate to eza

### PR DESCRIPTION
**Pull request by ronitnallagatla**
[exa is unmaintained, migrate to eza](https://github.com/ogham/exa/commit/fb05c421ae98e076989eb6e8b1bcf42c07c1d0fe)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
 
 ```sh
 # Install plugin
-plug "zap-zsh/exa"
+plug "zap-zsh/eza"
 ```
 
 ## ⚠️ Special Attention
-As mentioned in [this issue](https://github.com/zap-zsh/exa/issues/3), if you are using [zap-zsh/supercharge](https://github.com/zap-zsh/supercharge) you need to load this plugin after *superchage*, otherwise this plugin won't work because *supercharge* define the same `ls` alias (usefull to colorize `ls` output for who is not using **exa**).<BR><BR>
+As mentioned in [this issue](https://github.com/zap-zsh/eza/issues/3), if you are using [zap-zsh/supercharge](https://github.com/zap-zsh/supercharge) you need to load this plugin after *superchage*, otherwise this plugin won't work because *supercharge* define the same `ls` alias (usefull to colorize `ls` output for who is not using **eza**).<BR><BR>
```
Comment (mamaraddio) [inline README.md]: This line can't be accepted until the name (and so the URL) of this repository is changed.
Only @ChristianChiarulli can do it

File: eza.plugin.zsh [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    echo "ERROR: 'eza' command not found"
+    return
+fi
+
+# Create alias override commands using 'eza'
+alias ls='eza --group-directories-first --icons'
+
+# Use the --git flag if the installed version of eza supports git
+# Related to https://github.com/ogham/exa/issues/978
+if exa --version | grep -q '+git';
```
Comment (mamaraddio) [inline eza.plugin.zsh]: Maybe you forgot to change the executable name also in this line. Even if te eza installation provides a symlink to exa for backword compatibility change this to eza is a matte of consitency

**Review by mamaraddio — CHANGES_REQUESTED** [review]
Change exa to eza in line14 of `eza.plugins.zsh`

File: eza.plugin.zsh (hunk 2 above)
Comment (ronitnallagatla) [reply eza.plugin.zsh]: Thanks for catching that

**Review by mamaraddio — APPROVED** [review]

**Pull request merged**